app/src/main/java/io/github/muntashirakon/AppManager/main/MainViewModel.java

```diff
@@ -374,7 +374,7 @@ private void loadRunningApps() {
                 List<ProcessEntry> processEntries = (List<ProcessEntry>) IPCUtils.getServiceSafe().getRunningProcesses();
                 List<String> processNames = new ArrayList<>();
                 for (ProcessEntry entry : processEntries) {
-                    processNames.add(entry.name);
+                    processNames.add(entry.name.contains(":") ? entry.name.split(":")[0] : entry.name);
```

Comment:
Make the following method public and reuse it here instead:

https://github.com/MuntashirAkon/AppManager/blob/1040946cc61bc16db1d6aa4f4987fc5622a34c8c/app/src/main/java/io/github/muntashirakon/AppManager/runningapps/ProcessParser.java#L167-L173

Reply:
84ec85c2f1a81e51577ff3e8b100b3be92125073 fixed it like your suggestion.